File: silver/11399/c++/main.cpp
#include <iostream>
#include <algorithm>

int main()
{
    int n, data_sum = 0, result = 0;;
    std::cin >> n;
    int array[n];
    for (int i = 0; i < n; i++) {
        std::cin >> array[i];
    }
    
    std::sort(array, array+n);

    for (int data : array) {
        data_sum += data;
        result += data_sum;
    }
    std::cout << result;
    return 0;

}

File: silver/11399/python/main.py

if (__name__ == "__main__"):
    n = int(input())
    array = list(map(int, input().split()))
    array.sort()

    result, data_count = (0, 0)
    for data in array:
        data_count += data
        result += data_count
    print(result)